test/test_train_imagenet.py

```diff
@@ -163,9 +162,9 @@ def test_loop_fn(model, loader, device, context):
       correct += pred.eq(target.view_as(pred)).sum().item()
       total_samples += data.size()[0]
 
-    print('[{}] Accuracy={:.2f}%'.format(device,
-                                         100.0 * correct / total_samples))
-    return correct / total_samples
+    accuracy = 100.0 * correct / total_samples
+    test_utils.print_test_update(str(device), accuracy)
```

Comment:
Better not doing explicit conversion. Things added here must have implicit string conversion, as `format()` uses that.

Reply:
Switched to pass the device object instead of str'ing it